ui/src/components/Inventory/TextAnchorList.vue

```diff
@@ -1,47 +1,56 @@
 <template>
-  <ul class="anchor-list">
-    <li>
-      <label>{{ label.profile }}: </label>
-      <a :href="anchor.profileLink">{{ anchor.profileValue }}</a>
+  <ul class="text-anchor-list">
+    <li data-test="profile">
+      <label :for="label.profile">{{ label.profile }}: </label>
+      <a v-if="anchor.profileLink" @click="goto(anchor.profileLink)" :id="label.profile">{{ anchor?.profileValue }}</a>
+      <span v-else :id="label.profile">{{ anchor.profileValue }}</span>
     </li>
-    <li>
-      <label>{{ label.location }}: </label>
-      <a :href="anchor.locationLink">{{ anchor.locationValue }}</a>
+    <li data-test="location">
+      <label :for="label.location">{{ label.location }}: </label>
+      <a v-if="anchor.locationLink" @click="goto(anchor.locationLink)" :id="label.location">{{ anchor?.locationValue }}</a>
+      <span v-else :id="label.location">{{ anchor.locationValue }}</span>
     </li>
-    <li>
-      <label>{{ label.ipInterface }}: </label>
-      <a :href="anchor.ipInterfaceLink">{{ anchor.ipInterfaceValue }}</a>
+    <li data-test="ip-address">
+      <label :for="label.ipAddress">{{ label.ipAddress }}: </label>
+      <a v-if="anchor.ipAddressLink" @click="goto(anchor.ipAddressLink)" :id="label.ipAddress">{{ anchor.ipAddressValue }}</a>
+      <span v-else :id="(label.ipAddress)">{{ anchor.ipAddressValue }}</span>
     </li>
-    <li>
-      <label>{{ label.tag }}: </label>
-      <a :href="anchor.tagLink">{{ anchor.tagValue }}</a>
+    <li data-test="tag">
+      <label :for="label.tag">{{ label.tag }}: </label>
+      <a v-if="anchor.tagLink" @click="goto(anchor.tagLink)" :id="label.tag">{{ anchor.tagValue }}</a>
+      <span v-else :id="label.tag">{{ anchor.tagValue }}</span>
     </li>
   </ul>
 </template>
 
 <script lang="ts" setup>
-import { PropType } from 'vue'
 import { Anchor } from '@/types/inventory'
+import router from '@/router'
 
+defineProps<{
+  anchor: Anchor
+}>()
 
-defineProps({
-  anchor: {
-    type: Object as PropType<Anchor>,
-    required: true
-  }
-})
+const goto = (path: string | undefined) => {
+  if(!path) return
+  
+  router.push({
+    path
+  })
+}
 
 const label = {
   profile: 'Monitoring Profile',
   location: 'Monitoring Location',
-  ipInterface: 'IP Interface',
+  ipAddress: 'IP Address',
```

Comment:
```suggestion
  ipAddress: 'Management IP',
```
If I recall from the last meeting, this should be changed for clarity